src/System.Threading.Thread/tests/System.Threading.Thread.Tests.csproj

```diff
@@ -27,6 +27,7 @@
     <ProjectReference Include="..\pkg\System.Threading.Thread.pkgproj">
       <OSGroup>$(InputOSGroup)</OSGroup>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Threading\tests\System.Threading.Tests.csproj" />
```

Comment:
I moved some thread test helpers that I had previously added in S.T.T.Tests into S.T.Tests, reusing those from the S.T.Tests project. They're in ThreadTestHelpers.cs.